### PR TITLE
Fix/lazy json parse

### PR DESCRIPTION
**Pull request by brunoabreu**
#### What does this PR do? 

It fixes lazy JSON parse. Also changes how we set the global variables making it more generic.

#### How to test it?

You can see it working here: https://brunoh--carrefourbr.myvtex.com/Celulares-Smartphones-e-Smartwatches/Smartphones?crfimt=hm-tlink%7Ccarrefour%7Cmenu%7Ccampanha%7Csmartphones%7C1%7C120820

#### Related to / Depends on \*

Depends on https://github.com/vtex/render-server/pull/662

### CODE REVIEW

**Review by rafaelbac — APPROVED** [review]
LGTM

File: react/start/runtime.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
 const getValue = (element: HTMLTemplateElement) => {
   if (typeof element.content === 'undefined') {
-    return element.innerHTML
+    return element.textContent as string
   }
 
   // Using nodeValue instead of innerHTML because it's faster
```
Comment (lbebber) [inline react/start/runtime.ts:6]:
```suggestion
  // Using textContent instead of innerHTML because it's faster
```

**Review by lbebber — APPROVED** [review]
just 1 outdated comment fix

File: react/start/runtime.ts (hunk 1 above)
Comment (brunoabreu) [reply react/start/runtime.ts]: 👍

**Review by lbebber — APPROVED** [review]

**Pull request merged**